models/Article.php

```diff
@@ -106,108 +83,110 @@ function get_post(){
         ");
 
         $result = $req->fetchObject();
+        
         return $result;
     }
 
     // Fonction permettant d'éditer un article 
     function edit($title,$content,$posted,$id){
-        global $db;
-        $db = getPdo();
-
-        $e = [
+        $tableau = [
             'title'     => $title,
             'content'   => $content,
             'posted'    => $posted,
             'id'        => $id
         ];
 
         $sql = "UPDATE articles SET title=:title, content=:content, date=NOW(), posted=:posted WHERE id=:id";
-        $req = $db->prepare($sql);
-        $req->execute($e);
+        $req = $this->db->prepare($sql);
+        $req->execute($tableau);
     }
 
     // Fonction permettant d'insérer un nouvel article dans la base de données
     function post($title,$content,$posted){
-        global $db;
-        global $id;
-        $db = getPdo();
-
-        $p = [
+        $tableau = [
             'title'     =>  $title,
             'content'   =>  $content,
             'writer'    =>  $_SESSION['admin'],
             'posted'    =>  $posted
-
         ];
 
         $sql = "
         INSERT INTO articles(title, slug, introduction, content, writer, date, posted, image)
         VALUES(:title, '', '', :content,:writer,NOW(),:posted, '')
         ";
 
-        $req = $db->prepare($sql);
-        $req->execute($p);
-
-        $id = $db->lastInsertId(); //On doit mettre la fonction lastInsertId seulement après une fonction INSERT.
+        $req = $this->db->prepare($sql);
+        $req->execute($tableau);
     }
 
     // Fonction permettant d'insérer une nouvelle image dans la base de données
     function post_img($tmp_name, $extension){
-        global $db;
-        global $id;
-        $db = getPdo();
+        $id = $this->db->lastInsertId(); //On doit mettre la fonction lastInsertId seulement après une fonction INSERT.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5349276)